apps/google-analytics-4/lambda/serverless.yml

```diff
@@ -1,26 +1,107 @@
 service: google-analytics
 frameworkVersion: '3'
 
+# variablesResolutionMode: 20210326 # only needed for framework v >3
+
 plugins:
+  - serverless-domain-manager
   - serverless-offline
 
 custom:
-  environment: ${file(./config/serverless-env.${opt:stage, 'dev'}.yml)}
+  signingSecret:
+    prd: ${ssm:/aws/reference/secretsmanager/ci/apps/ga4-build-action/prd/signing-secret}
+    test: ${ssm:/aws/reference/secretsmanager/ci/apps/ga4-build-action/test/signing-secret}
+  appIdentityKey:
+    prd: ${ssm:/aws/reference/secretsmanager/ci/apps/ga4-build-action/prd/app-identity-key}
+    test: ${ssm:/aws/reference/secretsmanager/ci/apps/ga4-build-action/test/app-identity-key}
+  domainName:
+    prd: ga4-build.ctfapps.net
+    test: ga4-build-test.ctfapps.net
+  appDefinitionId:
+    prd: 5eHTUt9pILTGjYk3VFE9ta
+    test: yourTestId
+  customDomain:
+    domainName: ${self:custom.domainName.${env:STAGE, 'test'}}
+    stage: ${env:STAGE, 'test'}
+    createRoute53Record: true
+    endpointType: 'edge'
+    securityPolicy: tls_1_2
+  frontendUrl: 
+    prd: 'https://ga4.ctfapps.net/index.html'
+    test: 'http://localhost:3000'
   serverless-offline:
     httpPort: 8080
 
 provider:
   name: aws
-  stage: ${opt:stage, 'dev'}
   runtime: nodejs18.x
+  stage: ${env:STAGE, 'test'}
+  region: 'us-east-1'
+  timeout: 30
+  # lambdaHashingVersion: 20201221  # only needed for framework v >3
+  deploymentBucket:
+    name: cf-apps-serverless-deployment
```

Comment:
We don't actually know what this (presumable) s3 bucket is, or what it does yet. AWS access should hopefully make it clear if we need it and how it works